web/wp-content/themes/dctx-v2/inc/scripts.php

```diff
@@ -0,0 +1,154 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-ml7lug.php, but found phpcs-scan-ml7LuG.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).